c2corg_ui/static/partials/contexthelp/associate.html

```diff
@@ -0,0 +1,13 @@
+<p translate>Associate documents (e.g. routes, waypoints, books...etc)</p>
+<ul>
+<li><span translate> Enter a discriminatory part of the document's name. It is possible to search on multiple keywords. For a route, this includes the name of the summit(s) associated to the route.</span></li>
```

Comment:
Please remove the leading white space in this text to translate (ditto for the other lines): it is useless and I'm afraid it might cause problem while being extracted to Transifex, with the white space being there but not visible, or being not correctly extracted etc. I think it is really safer to remove it.